docs/integrations_api.yml

```diff
@@ -319,32 +319,32 @@ paths:
           type: string
         - name: firmware
```

Comment:
We could.
There is also paragraph:
"List known firmware artifacts"
Should we change this to:
"List known artifacts"?